Cargo.toml

```diff
@@ -38,10 +38,11 @@ debug = true
 
 [dependencies]
 fixedbitset = { version = "0.4.0", default-features = false }
-indexmap = "2.0"
+indexmap = { version = "2.0", default-features = true }
```

Comment:
What's the point of this? Defaults are on by default. :wink:

Reply:
D'oh 😣 chalk it up to inexperience with Cargo

Reply:
Agreed with the suggestion,
```suggestion
indexmap = "2.0"
```

Reply:
Fixed this. At the time I misunderstood how feature flags worked.